packages/mobile/src/backup/BackupPhraseContainer.tsx

```diff
@@ -155,6 +155,7 @@ const styles = StyleSheet.create({
     minHeight: 125,
     padding: 14,
     paddingTop: 16,
+    textAlignVertical: 'top',
```

Comment:
Before merging this, please ensure this looks right on both iOS and Android. I believe I fixed a bug with backup phrase styling on iOS a week or two ago and I'm worried this may regress it.